server/internal/adapter/integration/api_asset.go

```diff
@@ -0,0 +1,196 @@
+	"github.com/reearth/reearth-cms/server/internal/usecase/interfaces"
+	"github.com/reearth/reearth-cms/server/pkg/asset"
+	"github.com/reearth/reearth-cms/server/pkg/file"
+	"github.com/reearth/reearth-cms/server/pkg/id"
+	"github.com/reearth/reearth-cms/server/pkg/integrationapi"
+	"github.com/reearth/reearthx/usecasex"
+	"github.com/samber/lo"
+)
+
+func (s Server) AssetFilter(ctx context.Context, request AssetFilterRequestObject) (AssetFilterResponseObject, error) {
+	op := adapter.Operator(ctx)
+	uc := adapter.Usecases(ctx)
+
+	var sort asset.SortType
+	if request.Params.Sort != nil {
+		var err error
+		sort, err = asset.SortTypeFromString(string(*request.Params.Sort))
+		if err != nil {
+			return nil, err
+		}
+	}
+
+	f := interfaces.AssetFilter{
+		Keyword: nil,
+		Sort:    &sort,
+		Pagination: &usecasex.Pagination{
+			Before: nil,
+			After:  nil,
+			First:  lo.ToPtr(1000),
+			Last:   nil,
+		},
+	}
+
+	assets, pi, err := uc.Asset.FindByProject(ctx, id.ProjectID(request.ProjectId), f, op)
+	if err != nil {
+
+	}
+	itemList := lo.Map(assets, func(a *asset.Asset, _ int) integrationapi.Asset {
+		aa, err := ToAsset(a, uc.Asset.GetURL(a))
+		if err != nil {
+			return integrationapi.Asset{}
+		}
+		return *aa
+	})
+
+	return AssetFilter200JSONResponse{
+		Items:      &itemList,
+		Page:       lo.ToPtr(1),
+		PerPage:    lo.ToPtr(1000),
+		TotalCount: lo.ToPtr(pi.TotalCount),
+	}, nil
+}
+
+func (s Server) AssetCreate(ctx context.Context, request AssetCreateRequestObject) (AssetCreateResponseObject, error) {
+	uc := adapter.Usecases(ctx)
+	op := adapter.Operator(ctx)
+
+	f, err := ToFile(request.Body)
+	if err != nil {
+		return AssetCreate400Response{}, err
+	}
+
+	cp := interfaces.CreateAssetParam{
+		ProjectID: id.ProjectID(request.ProjectId),
+		File:      f,
+	}
+	a, err := uc.Asset.Create(ctx, cp, op)
+	if err != nil {
+		return AssetCreate400Response{}, err
+	}
+	aa, err := ToAsset(a, uc.Asset.GetURL(a))
+	if err != nil {
+		return AssetCreate400Response{}, err
+	}
+	return AssetCreate200JSONResponse(*aa), nil
+}
+
+func (s Server) AssetDelete(ctx context.Context, request AssetDeleteRequestObject) (AssetDeleteResponseObject, error) {
+	uc := adapter.Usecases(ctx)
+	op := adapter.Operator(ctx)
+	aId, err := uc.Asset.Delete(ctx, id.AssetID(request.AssetId), op)
+	if err != nil {
+		return AssetDelete400Response{}, nil
+	}
+	return AssetDelete200JSONResponse{
+		Id: &aId,
+	}, nil
+}
+
+func (s Server) AssetGet(ctx context.Context, request AssetGetRequestObject) (AssetGetResponseObject, error) {
+	uc := adapter.Usecases(ctx)
+	op := adapter.Operator(ctx)
+
+	a, err := uc.Asset.FindByID(ctx, id.AssetID(request.AssetId), op)
+	if err != nil {
+		return AssetGet400Response{}, err
+	}
+	aa, err := ToAsset(a, uc.Asset.GetURL(a))
+	if err != nil {
+		return AssetGet400Response{}, err
+	}
+	return AssetGet200JSONResponse(*aa), nil
+}
+
+func ToFile(multipartReader *multipart.Reader) (*file.File, error) {
```

Comment:
```suggestion
func toFile(multipartReader *multipart.Reader) (*file.File, error) {
```

toFile, toAsset, toPreviewType should be moved to another (e.g. `convert.go`) file. Do not make any one file too long.